BEIMA.Client/src/pages/DeviceTypes/DeviceTypesPage.js

```diff
@@ -13,34 +14,19 @@ const DeviceTypesPage = () => {
     setPageName('Device Types')
   },[setPageName])
  
-  const mockCall = async () => {
-    const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
-    await sleep(1000)
-    let data = []
-    for(let i = 0; i < 5; i++){
-      data.push({
-        deviceTypeId: i,
-        name: `Test Item Type #${i}`,
-        description: "A short informational description",
-        numDevices: i+4,
-      })
-    }
+  const DevicesCall = async () => {
+    let deviceTypeData = await GetDeviceTypeList();
+    
     // Map data into format supported by list
-    let mapped = data.map(({deviceTypeId, name, description, numDevices}) => {
-      return {
-        id: deviceTypeId,
-        name: name,
-        description: description,
-        numDevices: numDevices
-      }
-    })
-    return mapped
+    let data = deviceTypeData.response.map((item) => { return { id: item.id, name: item.name, description: item.description, notes: item.notes, lastModified: item.lastModified, deviceTag: "item types don't have a tag", buildingName: "building name needed", numDevices: "count at DB or remove? counting length on the DT return gives count of device type"} });
```

Comment:
I'm not sure that we need the deviceTag attribute on this?

Reply:
Same, the itemList.js that it's using is more appropriate for devices rather than device types. it might need some further revamping, or another file created.
Most of what was done here was proof it could use the API calls to get the data and display it once obtained.
I assume further refinement will happen to perfect it.

Reply:
> Same, the itemList.js that it's using is more appropriate for devices rather than device types. it might need some further revamping, or another file created. Most of what was done here was proof it could use the API calls to get the data and display it once obtained. I assume further refinement will happen to perfect it.

I left some comments regarding further refinement. I'm fine if we make another task to come back to that refinement and then get this PR through